packages/dds/map/src/directory.ts

```diff
@@ -739,7 +744,8 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
         if (message.type === MessageType.Operation) {
             const op: IDirectoryOperation = message.contents as IDirectoryOperation;
             if (this.messageHandlers.has(op.type)) {
```

Comment:
Similar here would be reasonable to switch to an assert

Reply:
done